client/app/dashboard/dashboard.controller.js

```diff
@@ -1,4 +1,8 @@
 angular.module('OpenEXP')
-    .controller('DashboardCtrl', ['$scope', ($scope) => {
+    .controller('DashboardCtrl', ['$scope', 'boardFactory', 'simulatorFactory', ($scope, boardFactory, simulatorFactory) => {
```

Comment:
ah ok, i'm aware of this as a little trick for minification, thanks for the heads up